pkg/registry/policy/poddisruptionbudget/strategy.go

```diff
@@ -91,9 +91,10 @@ func (podDisruptionBudgetStrategy) ValidateUpdate(ctx api.Context, obj, old runt
 	return append(validationErrorList, updateErrorList...)
 }
 
-// AllowUnconditionalUpdate is the default update policy for PodDisruptionBudget objects.
+// AllowUnconditionalUpdate is the default update policy for PodDisruptionBudget objects. Status update should
```

Comment:
Note that I am fine with this change but it doesn't solve the issue that is solved by Generation/ObservedGeneration.